wireguard-0.0.20190123 seems to be broken with 5.1 kernel, too:

```
copying path '/nix/store/11sc1qdp5zw27k2836c2l8mvvd330a3f-linux-5.1' from 'https://cache.nixos.org'...
copying path '/nix/store/07bp90bwm8gc8sxk0y54h42izk981xv3-source' from 'https://cache.nixos.org'...
copying path '/nix/store/hah16zwyf4glgijym2acrrxx378nac56-linux-5.1-dev' from 'https://cache.nixos.org'...
unpacking sources
unpacking source archive /nix/store/07bp90bwm8gc8sxk0y54h42izk981xv3-source
source root is source
patching sources
configuring
no configure script, doing nothing
building
  CC [M]  /build/source/src/main.o
In file included from <command-line>:0:0:
/build/source/src/compat/compat.h:783:10: fatal error: net/netfilter/nf_nat_core.h: No such file or directory
 #include <net/netfilter/nf_nat_core.h>
          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
make[4]: *** [/nix/store/hah16zwyf4glgijym2acrrxx378nac56-linux-5.1-dev/lib/modules/5.1.0/source/scripts/Makefile.build:276: /build/source/src/main.o] Error 1
make[3]: *** [/nix/store/hah16zwyf4glgijym2acrrxx378nac56-linux-5.1-dev/lib/modules/5.1.0/source/Makefile:1571: _module_/build/source/src] Error 2
make[2]: *** [Makefile:169: sub-make] Error 2
make[1]: *** [Makefile:15: __sub-make] Error 2
make: *** [Makefile:36: module] Error 2
builder for '/nix/store/xnz80bh68mij6ca8k9fkgrxa5z7s6rf0-wireguard-0.0.20190123.drv' failed with exit code 2
```

--- a/pkgs/os-specific/linux/wireguard/default.nix
+++ b/pkgs/os-specific/linux/wireguard/default.nix
@@ -21,7 +21,8 @@ stdenv.mkDerivation rec {
 
   nativeBuildInputs = [ perl ] ++ kernel.moduleBuildDependencies;
 
-  buildPhase = "make module";
+  buildFlags = [ "module" ];
+  installTargets = [ "module-install" ];
 
   meta = with stdenv.lib; {
     inherit (wireguard-tools.meta) homepage license maintainers;

--- a/pkgs/tools/networking/wireguard-tools/default.nix
+++ b/pkgs/tools/networking/wireguard-tools/default.nix
@@ -4,11 +4,11 @@ with stdenv.lib;
 
 stdenv.mkDerivation rec {
   name = "wireguard-tools-${version}";
-  version = "0.0.20190227";
+  version = "0.0.20190406";
 
   src = fetchzip {
     url = "https://git.zx2c4.com/WireGuard/snapshot/WireGuard-${version}.tar.xz";
-    sha256 = "1kaq0mjcdr4hxr3rx8z11rvlh4ycjsn5654fjw1gh1fdbjgczax5";
+    sha256 = "1rqyyyx7j41vpp4jigagqs2qdyfngh15y48ghdqfrkv7v93vwdak";
   };
 
   sourceRoot = "source/src/tools";

--- a/pkgs/tools/networking/wireguard-tools/default.nix
+++ b/pkgs/tools/networking/wireguard-tools/default.nix
@@ -4,11 +4,11 @@ with stdenv.lib;
 
 stdenv.mkDerivation rec {
   name = "wireguard-tools-${version}";
-  version = "0.0.20190123";
+  version = "0.0.20190227";
 
   src = fetchzip {
     url = "https://git.zx2c4.com/WireGuard/snapshot/WireGuard-${version}.tar.xz";
-    sha256 = "1lyl3nmsgp9jk9js3vz032vdx7cg9ynkwzdr19wrr26pkxhpcnxr";
+    sha256 = "1kaq0mjcdr4hxr3rx8z11rvlh4ycjsn5654fjw1gh1fdbjgczax5";
   };
 
   sourceRoot = "source/src/tools";
@@ -42,7 +42,7 @@ stdenv.mkDerivation rec {
     downloadPage = https://git.zx2c4.com/WireGuard/refs/;
     homepage = https://www.wireguard.com/;
     license = licenses.gpl2;
-    maintainers = with maintainers; [ ericsagnes mic92 zx2c4 ];
+    maintainers = with maintainers; [ elseym ericsagnes mic92 zx2c4 ];
     platforms = platforms.unix;
   };
 }

